moda-contracts/test/SplitsFactoryWithFork.t.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.21;
+
+import {SplitsFactory} from "../src/SplitsFactory.sol";
+import {ISplitsFactory} from "../src/interfaces/ISplitsFactory.sol";
+import {IManagement} from "../src/interfaces/IManagement.sol";
+import {ISplitMain} from "../src/interfaces/0xSplits/ISplitMain.sol";
+import {IOfficialModaContracts} from "../src/interfaces/IOfficialModaContracts.sol";
+import {SplitMainMock} from "./mocks/SplitMainMock.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "forge-std/Test.sol";
+import "forge-std/console2.sol";
+
+contract ModaRegistry is IOfficialModaContracts {
+    address _treasury;
+    uint32 _fee;
+    uint32 _feeMultiplier;
+
+    constructor(address treasury, uint32 fee, uint32 feeMultiplier) {
+        _treasury = treasury;
+        _fee = fee;
+        _feeMultiplier = feeMultiplier;
+    }
+
+    function getTreasuryInfo() external view returns (address, uint32, uint32) {
+        return (_treasury, _fee, _feeMultiplier);
+    }
+
+    function getSplitsFactory() external pure returns (ISplitsFactory) {
+        return ISplitsFactory(address(0));
+    }
+
+    function getManagement() external pure returns (IManagement) {
+        return IManagement(address(0));
+    }
+}
+
+contract SplitsFactoryWithForkTest is Test {
+    SplitsFactory _splitsFactory;
+    ModaRegistry _modaRegistry;
+    ISplitMain _splitMain;
+
+    string EVM_RPC_URL = vm.envString("EVM_RPC_URL");
+    address _splitMainAddress = 0x2ed6c4B5dA6378c7897AC67Ba9e43102Feb694EE;
+    address _payer = makeAddr("MODA_payer");
+    address _modaTreasury = makeAddr("MODA_treasury");
+    address _deployer = makeAddr("MODA_deployer");
+    address _alice = makeAddr("MODA_user_alice");
+    address _bob = makeAddr("MODA_user_bob");
+    address _distributor = makeAddr("MODA_distributor");
+
+    address _split;
+
+    uint32 _modaFee = 1_000;
+    uint32 _modaFeeMultiplier = 10_000;
+
+    address[] _beneficiaries;
+
+    event SplitCreated(address indexed sender, address indexed split);
+
+    function _sortAddresses(address[] memory addresses) internal pure returns (address[] memory) {
```

Comment:
SplitsFactory defines `_sortAddresses`. Why are we doing the same in this test? I forget what we said last time. It seems to me that this test is a false positive. We are testing code in the test, not code in the contract we are supposed to be testing.

We could do this:

`address(0x1), address(0x2)`, etc...